sdk/tests/deqp/functional/gles3/es3fTransformFeedbackTests.js

```diff
@@ -895,9 +895,13 @@ goog.scope(function() {
 
             if (linkFail) {
                 if (!es3fTransformFeedbackTests.isProgramSupported(this.m_progSpec, this.m_bufferMode)) {
-                    throw new Error('Not Supported. Implementation limits exceeded.');
+                    var msg = 'Not Supported. Implementation limits exceeded.';
+                    checkMessage(false, msg);
+                    throw new TestFailedException(msg);
```

Comment:
Does this cause the test runner to report a test failure? It shouldn't -- the C++ version of the test harness considers this case as being "not supported" rather than a failure. In the WebGL version I'm not sure it's necessary to distinguish between "passing" and "not supported", but "not supported" should not be equivalent to a failure.